section3/set10/challengeMoodly/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	args := os.Args[1:]
+
+	// if no user input received, then print usage msg
+	if len(os.Args) != 2 || args[0] == "" {
+		fmt.Println("[your name]")
+		return
+	}
+
+	// error handling (ex: user types in int)
+	n, err := strconv.Atoi(args[0])
+	if err == nil {
+		fmt.Printf("%d is not a name.\n", n)
```

Comment:
I think it should return after this statement

Reply:
added and fixed